aiplatform/src/main/java/aiplatform/CreatePipelineJobEmbeddingModelTuningSample.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package aiplatform;
+
+// [START aiplatform_sdk_embedding_model_tuning]
```

Comment:
As lee1premium mentioned there are quite a few aiplatform samples that include the complete class, including the main method

https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/aiplatform/src/main/java/aiplatform/BatchCreateFeaturesSample.java

https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/aiplatform/src/main/java/aiplatform/CreateBatchPredictionJobBigquerySample.java

I feel it is debatable which one is better to include in the documentation, but at least the current practice is consistent with the rest of the samples.    Do we have a new policy on this and plan to migrate all samples to it?

Reply:
Yes, many legacy code samples enclose the whole file in region tags. It is incorrect. See https://googlecloudplatform.github.io/samples-style-guide/ for guidelines.
Following incorrect implementation is rarely an argument in new implementations. In this particular case the subject in question is to enclose in region tags two areas instead of one.

I am unsure what you consider as "debatable" here. The provided reference to guidelines should be comprehensive enough.